src/cmds/scripts/win_postinstall.py

```diff
@@ -0,0 +1,632 @@
+import sys
+import getopt
+import os
+import subprocess
+import socket
+from tempfile import mkstemp
+from shutil import move
+import time
+
+install_path = ''
+pbs_path = ''
+
+
+def replace(file_path, pattern, subst):
+    fh, abs_path = mkstemp()
+    with os.fdopen(fh, 'w') as new_file:
+        with open(file_path) as old_file:
+            for line in old_file:
+                new_file.write(line.replace(pattern, subst))
+    # Remove original file
+    os.remove(file_path)
+    # Move new file
+    move(abs_path, file_path)
+
+
+def ValidateCredentials(username, password):
+    if (username != '' and password != ''):
```

Comment:
I agree with @hirenvadalia.  While the code with and without the parentheses might be the same, it is a question of python style.  Python style does not have parentheses outside of their if statements.  It's "if cond:" not "if (cond):".  The latter is more C style.  I disagree that it makes it any clearer to the reader what you are doing.  PEP doesn't cover the use of parentheses in an if statement.  I did find Google's python style guide and it says to use parentheses sparingly: https://google.github.io/styleguide/pyguide.html#Parentheses